src/Symfony/Component/Serializer/NameConverter/AdvancedNameConverterInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\NameConverter;
+
+/**
+ * Gives access to the format and the context in the property name converters.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface AdvancedNameConverterInterface extends NameConverterInterface
```

Comment:
where is this used?

Reply:
Nowhere, it's to be used in userland projects, we don't have this need in Symfony directly.

Reply:
I added a test to highlight the expected usage.